CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 @TotalDwarf03
+@evisos-ons
```

Comment:
Technically I don't think you should need this (likewise Kieran's username) as you are in the keh-dev team that is mentioned, happy for it to stay though.